src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -199,7 +199,7 @@ TEST_F(ApplicationManagerImplTest,
     }
   }
 
-  EXPECT_CALL(mock_hmi_mess_handler, SendMessageToHMI(_)).Times(AtLeast(0));
+  EXPECT_CALL(mock_hmi_mess_handler_, SendMessageToHMI(_)).Times(AtLeast(0));
```

Comment:
@pvvasilev I have no idea. I found that without it tests run OK. So I have to check and will remove it.